bin/removeNPMAbsolutePaths

```diff
@@ -14,11 +14,17 @@ const args = process.argv.slice(2);
 const folder = args[0];
 const opts = {
   force: false,
+  userSpecifiedKeys: false,
+  keys: [],
 };
 
 args.slice(1).forEach((arg) => {
```

Comment:
The argument parsing needs some work. This assumes that every single unmatched property is a key to be included.
As in: `removeNPMAbsolutePaths "MY_FOLDER" wrongPropertyThatShouldBeIgnoredButIsNot --force --userSpecific`

This should work so
`removeNPMAbsolutePaths "MY_FOLDER" --force --fields < ... ONLY STUFF HERE IS CONSIDERED AS KEYS ... >` is valid
`removeNPMAbsolutePaths "MY_FOLDER" --fields < ... ONLY STUFF HERE IS CONSIDERED AS KEYS ... > --force ` is valid and `--force` is not considered a property.

So basically stop parsing files as soon as it finds the next option starting with `--`.
This might mean that we can't use the `forEach` method and need a traditional for-loop. Not sure...